### DISCUSSION TITLE
GSOC Contributor Deadline

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#41] [Announcements]
Hello Everyone who is participating in GSOC this year. Please be sure to have your proposals in the GSOC portal by April 4 at 18:00 UTC; this is a deadline set by the program, and not one we can extend.

**Comment by CoToYo** [comment]
I submitted my proposal  3 days ago, but there is no one answering me or giving the feedback... So I just want to know whether the proposals submitted after March 31 will be reviewed? Thank you.

https://github.com/camicroscope/GSOC/discussions/38
https://github.com/camicroscope/GSOC/discussions/26
